crates/ty_python_semantic/resources/mdtest/narrow/conditionals/nested.md

```diff
@@ -222,6 +222,9 @@ g: str | None = "a"
 class A:
     x: str | None = None
 
+    def __bool__(self):
+        return self.x is not None
+
```

Comment:
Why add this? Tests pass without it. We don't assume anything about truthiness of class instances if they don't have a `__bool__` method, because we never know what subclass might.
```suggestion
```